app/react/Library/LibraryMap.js

```diff
@@ -7,7 +7,7 @@ import LibraryModeToggleButtons from 'app/Library/components/LibraryModeToggleBu
 
 export default class LibraryMap extends Library {
   static async requestState(requestParams, globalResources) {
-    return requestState(requestParams, globalResources, 'markers');
```

Comment:
So, was this actually passing `markers` as the value for `calculateTableColumns`? Something was wrong here to begin with.  Did you trace this problem when you debugged this issue?

Reply:
I think the `markers` parameter is legacy code that passed unnoticed since the files involved were not typed. This `markers ` parameter was passed to the `processQuery` function in `requestState` to set geolocation to true or false. Process query no longer handles geolocation on the query. 
Right now, on development, this triggers functionality for the table view when entering map view as well, with no side effects that I can tell.